features/dd-sdk-android-session-replay/src/main/kotlin/com/datadog/android/sessionreplay/internal/recorder/resources/Cache.kt

```diff
@@ -9,8 +9,8 @@ package com.datadog.android.sessionreplay.internal.recorder.resources
 internal interface Cache<K : Any, V : Any> {
 
     fun put(value: V) {}
-    fun put(element: K, value: V) {}
-    fun get(element: K): V? = null
+    fun put(key: String, value: V) {}
+    fun get(key: String): V? = null
```

Comment:
in this case the signature of the `Cache` key is misleading, because it states that key is parametrised by `K: Any`, which is not the case anymore with this change.